### PR TITLE
[kubeadm] Update CoreDNS Version

### LINKED ISSUE #1222: CoreDNS 1.1.3 not resolving ExternalName services
## Is this a BUG REPORT or FEATURE REQUEST?

Choose one: BUG REPORT

## Versions

**kubeadm version** (use `kubeadm version`): 1.11.4

**Environment**:
- **Kubernetes version** (use `kubectl version`): 1.11.4
- **Others**: CoreDNS 1.1.3

## What happened?

ExternalName services fail to resolve. 

## What you expected to happen?

ExternalName services to resolve without the need to manually update the CoreDNS version.

## How to reproduce it (as minimally and precisely as possible)?

Bring up a 1.11.4 cluster with kubeadm and create a service with ExternalName type. DNS resolution will fail for this service.

## Anything else we need to know?

Updating CoreDNS to v1.2.6 immediately resolved the issue.

### PR DESCRIPTION
**Pull request by detiber**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Resolves an issue that has been seen with ExternalName services not resolving.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Fixes kubernetes/kubeadm#1222

**Does this PR introduce a user-facing change?**:
```release-note
[kubeadm] Updates version of CoreDNS to 1.2.6
```

### CODE REVIEW

File: cmd/kubeadm/app/constants/constants.go [hunk 1]
```diff
@@ -309,7 +309,7 @@ const (
 	KubeDNSVersion = "1.14.13"
 
 	// CoreDNSVersion is the version of CoreDNS to be deployed if it is used
-	CoreDNSVersion = "1.2.4"
+	CoreDNSVersion = "1.2.6"
```
Comment (rajansandeep) [inline cmd/kubeadm/app/constants/constants.go]: The version in the [upgrade tests](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/upgrade/compute_test.go) also need to be changed.

**Comment by yagonobre** [comment]
Probably we need to update `cmd/kubeadm/app/phases/upgrade/compute_test.go` too

**Review by timothysc — APPROVED** [review]
/approve 

ping @MrHohn @rajansandeep for final LGTM

**Pull request merged**